src/js/client/ajaxCapable.js

```diff
@@ -62,18 +62,40 @@ A few more things to note:
         myTransaction.commit();
     };
 
+    // Strip null values to avoid validation errors with empty non-required fields.
+    // TODO:  Figure out why there are so many null values to strip
+    // TODO:  If this is required, add tests for it.
+    gpii.templates.ajaxCapable.stripNullValues = function (mapToStrip) {
```

Comment:
This function has already been removed in commits made in later branches.  I believe the specific commit was d6ad076a366ef1a0366491ee15680537b96dd1e9.  The newer version strips `null` values as a byproduct of using `fluid.model.transform`.   That work should be visible in either https://github.com/GPII/gpii-handlebars/pull/14 or https://github.com/GPII/gpii-handlebars/pull/15.